### ISSUE TITLE
The WebClipper popup doesn't show all its content

### ISSUE THREAD

**Issue by webformation** [laurent22/joplin#7618]
The pop-up of Web Clipper is larger than the available area, and because of that it "overflows". 

This post in the forum shows the problem : https://discourse.joplinapp.org/t/web-clipper-size-of-the-windows/28969/2

## Environment

Joplin version: 
Joplin 2.9.17 (prod, win32)
Version de Synchro : 3
Version du profil : 41
Trousseau supporté : Oui
Révision : a84a8e7

Platform:  Windows 11
OS specifics: Windows 11 Professionnel, 22H2, build 22623.1095

## Steps to reproduce

1. Start Web Clipper
2. Choose Clip Simplified Page
3. The footer of the popup is hidden as shown in this screenshot 
https://global.discourse-cdn.com/standard14/uploads/cozic/optimized/2X/a/a32230fb6a6b078e50e2a555b0935ef16a10eeda_2_1035x582.png

## Describe what you expected to happen

Ideally, the Web Clipper popup  should instead use a scrollbar, making the content fully available that way.

**Comment by Daeraxa** [comment]
I can't replicate this, Win10 - FF 108.0.2 using 2.8.1 of the clipper.

Could you update the issue with the clipper version and browser version.

![image](https://user-images.githubusercontent.com/58074586/212294147-54f9d2eb-6630-4a5d-bf10-3916590e0967.png)

**Comment by webformation** [comment]
Firefox : version 109
Web Clipper : 2.8.1
The problem arise with te recommanded size of the font size in the UI of Windows.
![image](https://user-images.githubusercontent.com/37415141/212299957-a28effef-34ec-46e0-b61d-c22ee604eb7a.png)

If I decreased the size, all the popup is visible, but my eyes cannot read the screen.

**Comment by tomasz1986** [comment]
This is easy to reproduce. Simply reduce your resolution while increasing DPI to a large value at the same time (e.g. 1920x1080 at 150% DPI). You may need to use custom DPI settings, because Windows may not allow large DPI values in the default dropdown on normal-sized desktop screens (but it will, for example, on a laptop).

The pop-up simply gets cut at some point with no access to what should be below.

![image](https://user-images.githubusercontent.com/5626656/212307036-09d85632-7e94-4439-860d-53735c8dfb17.png)

**Comment by yujit2003** [comment]
Please assign this to me

**Comment by Arghyahub** [comment]
Even at 150% DPI and 1920x1080 I'm unable to replicate the issue
![Screenshot 2023-02-05 011834](https://user-images.githubusercontent.com/84245432/216786904-eb1ce66b-e7ab-4dd6-a23c-5f8fc3780002.jpg)

**Comment by piyushyadav1617** [comment]
This appears to be resolved is it not? Does it occur for smaller screens?
![Screenshot (46)](https://user-images.githubusercontent.com/72244570/218043482-e37eb23a-b4fb-48c0-9f43-97712e1d6737.png)

**Comment by VaibhavAPatil** [comment]
I am **interested** in **this Issue**. Can I do **Pull Request** or you can **assign** this **issue to Me**.

I can reduce its height and add scrollbar to it

**Comment by sumit970** [comment]
hey! please assign this  issue to me

**Comment by B1TSH3LL** [comment]
i would like to work on this issue. please assign this to me. if it isn't solved!

**Comment by roman-r-m** [comment]
We do not assign issues but feel free to start working on it.

**Comment by huntermarchi** [comment]
ASSIGN IT TO ME I WILL FIX IT
